sound/soc/sof/intel/hda.h

```diff
@@ -471,6 +471,7 @@ struct sof_intel_hda_stream {
 	struct hdac_ext_stream hda_stream;
 	struct sof_intel_stream stream;
 	int host_reserved; /* reserve host DMA channel */
+	bool dmi_l1_compatible;
```

Comment:
Shall we add some comments to guide the user to set this for what kind of stream? e.g. "/* set false for capture stream support pause/release */" ?

Reply:
I'd leave this out. The logic when to set it might change later or vary from hw to another, so better to have it in implementation side only.